README.txt

```diff
@@ -1,9 +1,4 @@
 TEST
-Merge 2
-Merge Test
-Merge Tutorial
-Merge 3
-add Test2
-add Teest3
+This line is important
```

Comment:
This line is not important also, please remove it

Reply:
OK ~ I will do it

Reply:
Remove it